packages/@react-aria/dnd/intl/pt-PT.json

```diff
@@ -3,7 +3,7 @@
   "dragDescriptionTouch": "Faça duplo toque para começar a arrastar.",
   "dragDescriptionVirtual": "Clique para iniciar o arrasto.",
   "dragItem": "Arrastar {itemText}",
-  "dragSelectedItems": "Arrastar {count, plural, one {# item selecionado} outro {# itens selecionados}}",
+  "dragSelectedItems": "Arrastar {count, plural, one {# item selecionado} other {# itens selecionados}}",
```

Comment:
i think `outro` is correct here? https://translate.google.com/?sl=pt&tl=en&text=outro&op=translate&hl=en

Reply:
No, this should not have been translated. It's a keyword, not part of the message.